bin/runtime-common/src/refund_relayer_extension.rs

```diff
@@ -158,6 +159,14 @@ pub enum CallInfo {
 }
 
 impl CallInfo {
+	/// Returns true if call is a message delivery call (with optional finality calls).
+	fn is_message_delivery_call(&self) -> bool {
```

Comment:
Nit: I would define this method on `MessagesCallInfo`. But for consistency with the enum variant names, it would need to be called `is_receive_messages_proof` instead.

Reply:
renamed, but left here